NepovinnyUkolOOP/Obdelnik.cs

```diff
@@ -73,12 +73,12 @@ public Obdelnik(double strana) : this(strana, strana)
 
         public void Vypis()
         {
-            Console.WriteLine($"Sirka: {Sirka}, Vyska: {Vyska}, Obsah: {Obsah}, Obvod: {Obvod}");
+            Console.WriteLine($"Sirka: {Sirka}, Vyska: {Vyska}, Obsah: {Obsah} a Obvod: {Obvod}");
         }
-        public void Zvetsi(double zvysitSirka, double zvysitVyska)
+        public void Zvetsi(double zvysitSirku, double zvysitVysku)
```

Comment:
nit: Konvence je mit prazdnou mezeru mezi propertami/metodami
```suggestion

        public void Zvetsi(double zvysitSirku, double zvysitVysku)
```